eval.go

```diff
@@ -183,6 +184,15 @@ func (d *indexData) Search(ctx context.Context, q query.Q, opts *SearchOptions)
 	docCount := uint32(len(d.fileBranchMasks))
 	lastDoc := int(-1)
 
+	var curRepo uint16
+	var repoPriority float64
+	if len(d.repoMetaData) > 0 {
+		repoPriority, err = strconv.ParseFloat(d.repoMetaData[0].RawConfig["priority"], 64)
+		if err != nil {
```

Comment:
You can avoid the error check since repoPriority will be zero when err != nil.

Reply:
Are you sure? The documentation doesn't mention it and it is not obvious from the source code.

Reply:
Well, I guess now that you bring it up, I'm not sure sure it's guaranteed, but, in practice I've relied on that many times and it's a common thing to return the zero value of a return argument when there's an error.